python/tk_desktop/desktop_window.py

```diff
@@ -368,6 +375,27 @@ def handle_project_refresh_action(self):
         else:
             self._project_model._refresh_data()
 
+    def handle_advanced_project_setup_action(self):
+        """
+        Display the classic project setup wizard if the current
+        user appears to have sufficient permissions to actually
+        setup a project. If not, pop up an error dialog informing
+        them of the problem.
+        """
+        self.setup_project_widget.project = self.current_project
+
+        # The first time a user selects the Advanced project setup
+        # menu item, display the Setup Project help popup to provide
+        # more information about this feature.
+        wizard_setting = "advanced_project_setup_launched"
+        wizard_help_shown = self._load_setting(wizard_setting, False, False)
```

Comment:
Could you pass named parameters for he booleans? It would help the reader understand what those values are without having to scroll back to the documentation of the method.